docs/command/atlas-plugin-update.txt

```diff
@@ -0,0 +1,99 @@
+.. _atlas-plugin-update:
+
+===================
+atlas plugin update
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Update Atlas CLI plugin.
+
+Update an Atlas CLI plugin.
```

Comment:
Was it intentional to have this repeat twice?

Reply:
Yes that is intentional. Line 15 is the short description shown in the 'help' output and line 17 is the first line of the long message shown in the 'help <this-command>' output.